src/client/components/FavouriteList/FavouriteList.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { action } from '@storybook/addon-actions';
+import './FavouriteList.styles.css';
+
+export default function FavouriteList({ productList }) {
+  const [counter, setCounter] = useState(0);
+  const handleClick1 = () => {
+    setCounter(counter + 1);
+  };
+
+  const handleClick2 = () => {
+    setCounter(counter - 1);
+  };
+
```

Comment:
This component must be broken down into sub components.
See here https://reactjs.org/docs/thinking-in-react.html